e2e/provision/playbooks/roles/kubeadm/tasks/kubeadm-Debian.yml

```diff
@@ -0,0 +1,52 @@
+- name: Install kubelet kubeadm kubectl on {{ ansible_os_family }}
+  become: true
+  block:
+    - name: Install curl and apt-transport-https packages
+      ansible.builtin.package:
+        name:
+          - apt-transport-https
+          - curl
+          - libseccomp2
+        state: present
+
+    - name: Ensure /etc/apt/keyrings directory exists
+      ansible.builtin.file:
+        path: /etc/apt/keyrings
+        state: directory
+        mode: '0644'
+
+    - name: Add gpg key
+      ansible.builtin.get_url:
+        url: https://pkgs.k8s.io/core:/stable:/v{{ k8s_ver }}/deb/Release.key
+        dest: /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+        mode: '0644'
+        force: true
+
+    - name: Add specified repository into sources list using specified filename
+      ansible.builtin.apt_repository:
+        repo: deb [trusted=yes] https://pkgs.k8s.io/core:/stable:/v{{ k8s_ver }}/deb/ /
+        state: present
+        filename: kubernetes.list
+
+    - name: Install packages kubelet kubeadm and kubectl
+      ansible.builtin.package:
+        name:
+          - kubelet
+          - kubeadm
+          - kubectl
+        state: present
+
+    - name: Hold kubeadm
+      ansible.builtin.dpkg_selections:
+        name: kubeadm
+        selection: hold
+
+    - name: Hold kubelet
+      ansible.builtin.dpkg_selections:
+        name: kubelet
+        selection: hold
+
+    - name: Hold kubectl
+      ansible.builtin.dpkg_selections:
+        name: kubectl
+        selection: hold
```

Comment:
@dkosteck Is it possible to a task to configure `vm.mmap_rnd_bits` starting kernel 6.x they are 32bit and asn1 have issues when it is like that. 

https://github.com/actions/runner-images/issues/9491#issuecomment-1989718917

`sudo sysctl vm.mmap_rnd_bits=28`, earlier we had to do this for Ubuntu 22 on GCP which has the new 6.5-gcp kernel.

Reply:
By the way, the changes that this MR made, I don't think are tested in the CI.